docs/transcriptomics/metatranscriptomics.rst

```diff
@@ -0,0 +1,152 @@
+======================
+Metatranscriptomics
+======================
+
+Protocol provided by Guillem Salazar.
+
+-----------------------------------------
+The datasets
+-----------------------------------------
+RNA was extracted, processed and analyzed with a hybrid approach where both de-novo assembly of the data is used together with the use of the genome as reference.
+Out of this procedure, transcript abundance tables are produced where the number of transcripts is quantified for each gene in the sample. As the main goal of the present guide is to analyze the response of the native soil community, all genes from the inoculum were further excluded.
+Genes are finally assigned a metabolic function by using the KEGG database. All genes belonging to the same function are added up and thus the gene abundance table is then transformed into a functional abundance table. For example, the transcript abundance of all genes from different organisms that encode the pyruvate kinase (designated as K00873 in the KEGG database) are added up. In this way we quantify the abundance of each metabolic function in each sample (regardless of the microbe encoding for it).
+This functional abundance table and an additional table with the sample information are the two main ingredients for the following analysis.
+
+
+-----------------------------------------
+Loading and formatting the data
```

Comment:
Please add a list of all the packages used for the analysis with links to the documentation